src/pages/templates/pages/partial/dashboard_panel_popular_reads.html

```diff
@@ -1,6 +1,10 @@
 <div class="box box-mascot-tipped">
+    {% if is_teacher %}
     <h2>Student reading</h2>
     <p>See what your students are reading and what they said they learned.</p>
```

Comment:
This message is duplicated now.
<img width="571" alt="image" src="https://user-images.githubusercontent.com/2131184/182447842-9b1fa0be-1d4a-4802-89cd-981665fc95d6.png">

Reply:
Thanks for noticing the duplication.  The upper message was missed because `grep` did not find it.  While the gist is the same, the text is actually different. I've deleted it.